### DISCUSSION TITLE
QuickSearch does not work on Column.cs, only Row.cs

### DISCUSSION THREAD

**Discussion by minhhungit** [serenity-is/Serenity#6017] [General]
- latest version

**Comment by VictorTomaili** [comment]
It's in the `ListRequestHandler`, so the `ListRequestHandler` doesn't know the related `columns.cs`
[Related file](https://github.com/serenity-is/Serenity/blob/38f2b11d54da700810b8bd83dcc5d7f1c5562f6b/src/Serenity.Net.Services/RequestHandlers/List/ListRequestHandler.cs#L171)

**Reply by minhhungit** [reply]
thanks Victor

**Reply by minhhungit** [reply]
but I think Columns.cs relate to grid, so maybe ListRequestHandler should cascade read the attribute on columns => rows ? like QuickFilter

**Reply by VictorTomaili** [reply]
QuickFilters are generated on client-side, QuickSearch is used while getting data from the database, also columns data is not related to list request and one row can have multiple columns file. It's impossible to determine the correct columns file without sending it in request. And this is not a good practice. The list request shouldn't contain the columns information.

**Reply by minhhungit** [reply]
thank you for the information sir
